source/includes/api/_geostore.md

```diff
@@ -1,92 +1,112 @@
+
+## Overview of available endpoints
+
+The Geostore API consists of a number of endpoints to help users interact with geostore objects. Many of them are essentially helpers to make it easy to access geometries from useful data sources, such as:
+
+- **Administrative areas** (countries, regions, ect.) from the [Database of Global Administrative Areas (GADM)](https://en.wikipedia.org/wiki/GADM).
+- **Protected areas** (world heritage sites, national parks, ect.) from the [The World Database on Protected Areas](https://www.protectedplanet.net/c/world-database-on-protected-areas).
+- **Land activity/use types** (logging, mining, ect.) from the [WRI](https://www.wri.org/) [managed forest](https://data.globalforestwatch.org/datasets/managed-forest-concessions), [mining](https://data.globalforestwatch.org/datasets/mining-concessions), and [oil palm](https://data.globalforestwatch.org/datasets/20398d4dc36e47bd92b559786670f270_1) concessions, and [WWF](https://wwf.panda.org)/[RESOLVE](https://www.resolve.ngo/about.htm) [tiger conservation](https://data.globalforestwatch.org/datasets/tiger-conservation-landscapes) datasets.  
+
+As these data sets are periodically updated, these "helper" endpoints are updated to use the latest data. So unless you really need an older version of a data set, such as GADM, you should always use the latest endpoint version. See the table below to get an overview of what can be done; click on the description to get more details.
+
+> Comparison of endpoint versions
+
+| Version | Description       |
+|---------|------------------ |
+| `v1`    | GADM version 2.8. |
+| `v2`    | GADM version 3.6. |
+
+> Overview of endpoints
+
+| Method | Path                                | Description                                                                   |
+|--------|-------------------------------------|------------------------------------------------------------------------------ |
+| POST   | `/v1/geostore`                      | [Create a geostore object](#create-a-geostore-object)                         |
+| GET    | `/v2/geostore/:id`                  | [Get a geostore object by Geostore id](#get-a-geostore-object-by-geostore-id) |
+| POST   | `/v2/geostore/find-by-ids`          | [Get geostore objects by Geostore ids](#get-geostore-objects-by-geostore-ids) |
+| GET    | `/v2/geostore/:id/view`             | [Get a geostore object by Geostore id and view at GeoJSON.io](#get-a-geostore-object-by-geostore-id-and-view-at-geojson-io) |
+| GET    | `/v2/geostore/admin/list`           | [Get all Geostore ids, names and country codes](#get-all-geostore-ids-names-and-country-codes) |
+| GET    | `/v2/geostore/admin/:iso`           | [Get a geostore object by country code](#get-country-code)                    |
```

Comment:
Cant seem to find docs for this particular endpoint

Reply:
Link was incorrect, fixed